pkg/client/client_test.go

```diff
@@ -35,8 +35,10 @@ func TestCreate(t *testing.T) {
 		"publisher": "my-pub",
 	}
 
+	c := NewClient(http.DefaultClient)
```

Comment:
I think there's a few more tests to fix in here, so that the unit test pass again. could you fix those?

Reply:
Yeah, that's weird. I guess I didn't push up all the changes?